UXSDKBetaSampleApp/Podfile

```diff
@@ -1,10 +1,21 @@
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
This source isn't needed in the actual podfile unless you are redirecting from a previous source command.